deploy/LOCAL_INSTALL.md

```diff
@@ -130,6 +150,20 @@ This is expected to take a while
    docker-compose exec pipeline-runner download_reference_data.sh $BUILD_VERSION
    
    ``` 
+   Periodically, you may want to update the reference data in order to get the latest versions of these annotations. 
+To do this, run the following commands to update the data. All subsequently loaded data will then have the updated 
+annotations, but you will need to re-load previously loaded projects to get the updated annotations.
+   ```bash
+   BUILD_VERSION=38                 # can be 37 or 38
+   
+   # Update clinvar 
+   docker-compose exec pipeline-runner rm -rf "/seqr-reference-data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+   docker-compose exec pipeline-runner gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/clinvar/clinvar.GRCh${BUILD_VERSION}.ht" "/seqr-reference-data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+  
+   # Update all other reference data
+   docker-compose exec pipeline-runner rm -rf "/seqr-reference-data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
+   docker-compose exec pipeline-runner gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/all_reference_data/combined_reference_data_grch${BUILD_VERSION}.ht" "/seqr-reference-data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
```

Comment:
No `gsutil rsync` commands are required for the on-prem option.

Reply:
They need to rsync the content of our reference data bucket on GCP to their local machine